hugging_face/finetune_vista3d_for_hugging_face_pipeline.ipynb

```diff
@@ -0,0 +1,426 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.        numpy_output = np.squeeze(numpy_output, axis=0)
</pre><p>Just FYI we can use <code>SqueezeDim</code> in the MONAI here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1946/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
Thanks for this head-up. This function here is to transter the output to numpy for display purpose. I think the numpy squeeze should be fine.